tests/OBSLocal.py

```diff
@@ -73,12 +80,18 @@ def oscrc(userid):
             ]))
 
     def osc_user(self, userid):
+        print(f'setting osc user to {userid}')
```

Comment:
wow, when did python start to copy from ruby? This looks superuseful

Reply:
or perl/php/bash/etc, but yes